practice-app/server/src/api/controllers/event/createEvent.js

```diff
@@ -0,0 +1,98 @@
+import { Event } from '../../../models/index.js';
+import { validateEvent } from '../../validators/event.validator.js';
+import axios from 'axios';
+import { apiKeyGeo } from '../../../config/index.js';
+import mongoose from 'mongoose';
+import Lesson from '../../../models/lesson.js';
+
+async function getCoordinates(address) {
+
+    const http_formatted_address = address.replace(/\s/g, '+');
+
+    var url = "https://maps.googleapis.com/maps/api/geocode/json?";
+
+    var params = {
+        address: http_formatted_address,
+        key: apiKeyGeo
+    };
+
+    Object.keys(params).forEach(function (url_key) { url += "&" + url_key + "=" + params[url_key]; });
```

Comment:
We may make it like "{ url += url_key + "=" + params[url_key] + "&"; });" to start the query parameters without "&" at the beginning.